src/scales/scale.time.js

```diff
@@ -688,31 +688,19 @@ class TimeScale extends Scale {
 	}
 
 	/**
-	 * @private
+	 * @param {number} value - Milliseconds since epoch (1 January 1970 00:00:00 UTC)
 	 */
-	_getPixelForOffset(time) {
+	getPixelForValue(value) {
 		var me = this;
 		var offsets = me._offsets;
-		var pos = interpolate(me._table, 'time', time, 'pos');
+		var pos = interpolate(me._table, 'time', value, 'pos');
 		return me.getPixelForDecimal((offsets.start + pos) * offsets.factor);
 	}
 
-	getPixelForValue(value) {
-		var me = this;
-
-		if (typeof value !== 'number') {
```

Comment:
This is 1.7ms in uPlot. Could consider re-adding it, to make things easier for plugin developers. OTOH we have `_parse` that can be used (should probably make that a public one, but let's let it settle first).